src/Categories/Category/Instance/Simplex.agda

```diff
@@ -1,67 +1,152 @@
 
 Δ : Category 0ℓ 0ℓ 0ℓ
 Δ = record
-  { Obj       = ℕ
-  ; _⇒_       = λ m n → Σ (Fin m → Fin n) (λ f → _≤_ =[ f ]⇒ _≤_)
-  ; _≈_       = λ { (f , mf) (g , mg) → ∀ x → f x ≡ g x }
-  ; id        = idF , idF
-  ; _∘_       = λ { (f , mf) (g , mg) → f ∙ g , mf ∙ mg }
-  ; assoc     = λ _ → refl
-  ; sym-assoc = λ _ → refl
-  ; identityˡ = λ _ → refl
-  ; identityʳ = λ _ → refl
-  ; identity² = λ _ → refl
-  ; equiv     = record
-    { refl  = λ _ → refl
-    ; sym   = λ eq x → sym (eq x)
-    ; trans = λ eq₁ eq₂ x → trans (eq₁ x) (eq₂ x)
+  { Obj = ℕ
+  ; _⇒_ = _Δ⇒_
+  ; _≈_ = _≗_
+  ; id = ε
+  ; _∘_ = _⊚_
+  ; assoc = Δ-eq refl
+  ; sym-assoc = Δ-eq refl
+  ; identityˡ = Δ-eq refl
+  ; identityʳ = Δ-eq refl
+  ; identity² = Δ-eq refl
+  ; equiv = record
+    { refl = Δ-eq refl
+    ; sym = λ eq → Δ-eq (sym (Δ-pointwise eq))
+    ; trans = λ eq₁ eq₂ → Δ-eq (trans (Δ-pointwise eq₁) (Δ-pointwise eq₂))
     }
-  ; ∘-resp-≈  = λ {_ _ _ f g h i} eq₁ eq₂ x → trans (cong (λ t → proj₁ f t) (eq₂ x)) (eq₁ (proj₁ i x))
+  ; ∘-resp-≈ = λ {_ _ _ f g h i} eq₁ eq₂ → Δ-eq (trans (cong ⟦ f ⟧ (Δ-pointwise eq₂)) (Δ-pointwise eq₁))
   }
 
+
+-- For completeness, here are the aforementioned simplical identities. These may seem /slightly/
+-- different than the ones presented before, but it's just re-indexing to avoid having to use 'pred'.
 open Category Δ
 
---------------------------------------------------------------------------------
--- Face + Degeneracy Maps
-
-face-map : ∀ {n} → Fin (ℕ.suc n) → Fin n → Fin (ℕ.suc n)
-face-map Fin.zero    k           = Fin.suc k
-face-map (Fin.suc i) Fin.zero    = Fin.zero
-face-map (Fin.suc i) (Fin.suc k) = Fin.suc (face-map i k)
-
-face-mono : ∀ {n} → (i : Fin (ℕ.suc n)) → _≤_ =[ face-map i ]⇒ _≤_
-face-mono Fin.zero    {_}         {_}         le       = s≤s le
-face-mono (Fin.suc i) {Fin.zero}  {_}         _        = z≤n
-face-mono (Fin.suc i) {Fin.suc _} {Fin.suc _} (s≤s le) = s≤s (face-mono i le)
-
-face : ∀ {n} → Fin (ℕ.suc n) → n ⇒ ℕ.suc n
-face i = face-map i , face-mono i
-
-degeneracy-map : ∀ {n} → Fin n → Fin (ℕ.suc n) → Fin n
-degeneracy-map Fin.zero    Fin.zero    = Fin.zero
-degeneracy-map Fin.zero    (Fin.suc k) = k
-degeneracy-map (Fin.suc i) Fin.zero    = Fin.zero
-degeneracy-map (Fin.suc i) (Fin.suc k) = Fin.suc (degeneracy-map i k)
-
-degeneracy-mono : ∀ {n} → (i : Fin n) → _≤_ =[ degeneracy-map i ]⇒ _≤_
-degeneracy-mono Fin.zero    {Fin.zero}  {_}         _        = z≤n
-degeneracy-mono Fin.zero    {Fin.suc _} {Fin.suc _} (s≤s le) = le
-degeneracy-mono (Fin.suc i) {Fin.zero}  {_}         _        = z≤n
-degeneracy-mono (Fin.suc i) {Fin.suc _} {Fin.suc _} (s≤s le) = s≤s (degeneracy-mono i le)
-
-degeneracy : ∀ {n} → Fin n → ℕ.suc n ⇒ n
-degeneracy i = degeneracy-map i , degeneracy-mono i
+-- δᵢ ∘ δⱼ = δⱼ₊₁ ∘ δᵢ if i ≤ j
+face-comm : ∀ {n} {i j : Fin (suc n)}  → i ≤ j → δ (inject₁ i) ∘ δ j ≈ δ (suc j) ∘ δ i
+face-comm {_} {zero}  {j}     z≤n      = Δ-eq refl
```

Comment:
I think you should be able to split on `i ≤ j` directly, without the implicits and still get this?

Reply:
Same deal as above, some gnarly unification problem error.